### ISSUE TITLE
[NEWS] Zasób bidding/offers/{offerID}/bid jest od dziś w wersji public. Metodę doBidItem oznaczamy jako deprecated i wprowadzimy w niej zmiany / Bidding/offers/{offerID}/bid resource released in public version. doBidItem is marked as deprecated

### ISSUE THREAD

**Issue by PawelTaberski** [allegro/allegro-api#2090]
Zakończyliśmy okres testowy dla zasobu [bidding/offers/{offerID}/bid](https://developer.allegro.pl/documentation/#operation/placeBid), dzięki któremu złożysz ofertę kupna w trwającej licytacji oraz pobierzesz informację na ten temat. **Wersję beta będziemy wspierali do 23.08.2019.** W tym okresie możesz systematycznie przechodzić na wersję public. Aby przejść na nową wersję wystarczy zmienić w nagłówku:

‘application/vnd.allegro.beta.v1+json’ na ‘**application/vnd.allegro.public.v1+json**’.

Metodę [doBidItem](https://allegro.pl/webapi/documentation.php/show/id,382) już dziś oznaczamy jako [deprecated](https://allegro.pl/webapi/faq.php#faq_8) i dodatkowo:

- **od 2.09.2019** nie kupisz przy jej pomocy oferty w formacie Kup Teraz;
- **14.10.2019** usuniemy tą metodę.

----------------------------------------

We have completed the test period for [bidding/offers/{offerID}/bid](https://developer.allegro.pl/documentation/#operation/placeBid) which allows you to place a bid in an auction and receive data about your bid. **We will support the beta version until 23.08.2019.** During this period, you can gradually switch to the public version. To switch to the public version, you should change the header value ‘accept’ from:

‘application/vnd.allegro.beta.v1+json’ to ‘**application/vnd.allegro.public.v1+json**’.

We marked [doBidItem](https://allegro.pl/webapi/documentation.php/show/id,1138) method as [deprecated](https://allegro.pl/webapi/faq.php#faq_8) and:

- **from 2.09.2019** you will not be able to use it to buy in offer with BuyNow format;
- we will remove this method **on 14.10.2019**.

**Comment by A2lEX** [comment]
@PawelTaberski right now its impossible to buy "BuyNow" goods. am i correct? Will be this feature in future?

**Comment by PrzemyslawLukanowski** [comment]
@A2lEX Yes, You're right, it's impossible. We don’t now provide this functionality by REST API.

**Comment by zworo** [comment]
> @A2lEX Yes, You're right, it's impossible. We don’t now provide this functionality by REST API.

Czy wiadomo kiedy taka funkcjonalność zostanie dodana? Czy będzie to PRZED usunięciem metody doBidItem z WebAPI ?

**Comment by PrzemyslawLukanowski** [comment]
Na tę chwilę nie planujemy dodania takiego zasobu, uzasadnienie znajdziesz w wątku #1896.
